template/contracts/src/BuyMeACoffee.sol

```diff
@@ -47,7 +47,14 @@ contract BuyMeACoffee {
     error OnlyOwner();
 
     event BuyMeACoffeeEvent(address indexed buyer, uint256 price);
-    event NewMemo(address indexed userAddress, uint256 time, uint numCoffees, string userName, string twitterHandle, string message);
+    event NewMemo(
+        address indexed userAddress,
+        uint256 time,
+        uint256 numCoffees,
+        string userName,
```

Comment:
I kind of think we should stop ask the `userName` and the `twitterHandle` and instead use ENS data for this.

Reply:
100% with you on this